src/Billboard/IBillboardRegistry.sol

```diff
@@ -0,0 +1,145 @@
+//SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+interface IBillboardRegistry is IERC721 {
+    //////////////////////////////
+    /// Error
+    //////////////////////////////
+
+    error AdminNotFound();
+
+    error BoardNotFound();
+
+    error InvalidBoardId();
+
+    error InvalidAddress();
+
+    error OperatorNotFound();
+
+    error Unauthorized(string type_);
+
+    //////////////////////////////
+    /// Event
+    //////////////////////////////
+
+    event Mint(uint256 indexed tokenId_, address to_);
+
+    event Transfer(uint256 indexed tokenId_, address from_, address to_);
+
+    //////////////////////////////
+    /// Struct
+    //////////////////////////////
+
+    struct Board {
+        address owner;
```

Comment:
i was thinking that maybe `owner` is not a good name since it's easily confused with `token.ownerOf`.

Reply:
Rename owner to creator (better than minter).